test/stage-autocompleter.test.js

```diff
@@ -197,7 +197,7 @@ describe('StageAutoCompleter', () => {
 
           context('when the prefix begins with $', () => {
             context('when the token is on the same line', () => {
-              const completer = new StageAutoCompleter('4.4.0', textCompleter, fields, null);
+              const completer = new StageAutoCompleter('5.0.0', textCompleter, fields, null);
```

Comment:
Would this match the 5.0rc too?

Reply:
I would say that according to the current check (how it is currently implemented) no.

Reply:
Also, the interesting fact is that the keywords version check happens here in the `ace-autocompleter` module, but stages version check happens in the aggregation plugin itself:
https://github.com/mongodb-js/compass-aggregations/blob/master/src/components/stage-editor-toolbar/stage-operator-select.jsx#L68-L72

Also interesting that we check accumulators version for project stage only:
https://github.com/mongodb-js/ace-autocompleter/blob/master/lib/stage-autocompleter.js#L51-L64